app/templates/_base_page.html

```diff
@@ -1,48 +1,73 @@
+{% extends "template.njk" %}
 
 {# Import GOV.UK Frontend components for globale usage#}
 {% from "components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
 {% from "components/button/macro.njk" import govukButton %}
 {% from "components/phase-banner/macro.njk" import govukPhaseBanner %}
 {% from "components/tabs/macro.njk" import govukTabs %}
 
-{# This can be removed once we start using govuk-frontend template #}
-{# The reason calling this block here is to change the header title so that it does not
-   include the phase tag #}
-{% block inside_header %}
-  <div class="header-title">
-    <a href="/">Digital Marketplace</a>
-  </div>
-{% endblock %}
+{% set assetPath = '/admin/static' %}
 
-{# This can be removed once we start using govuk-frontend template #}
-{# The reason for calling this is to ensure we do not output the user research signup banner #}
-{% block after_header %}
+{% block head %}
+  {% include "layouts/_custom_dimensions.html" %}
+  {% include "layouts/_site_verification.html" %}
+  <link type="text/css" rel="stylesheet" href="{{ asset_fingerprinter.get_url('stylesheets/application.css') }}" />
 {% endblock %}
 
-{# override content block to add a beforeContent block like the one in govuk-frontend template #}
-{% block content %}
-  {% block top_header %}{% endblock %}
-  <div id="wrapper">
-    {% block beforeContent %}
-      {{ govukPhaseBanner({
-        "tag": {
-          "text": "beta"
-        },
-        "html": 'Help us improve the Digital Marketplace - <a class="govuk-link" href="'  + url_for('external.help') + '">send your feedback</a>'
-      }) }}
-      {% block breadcrumbs %}{% endblock %}
-    {% endblock %}
-
-    <main id="content" role="main">
-      {% include "toolkit/flash_messages.html" %}
-      {% block main_content %}
-      {% endblock %}
-    </main>
-  </div>
+{% block header %}
+
+  {% set headerNavigation = [
+    {
+      'text': 'Guidance',
+      'href': 'https://www.gov.uk/guidance/buying-and-selling-on-the-digital-marketplace'
+    },
+    {
+      'text': 'Help',
+      'href': url_for('external.help')
+    }
+  ]%}
+
+  {% set user_logged_in = logged_in|default(false, true) %}
+
+  {% if current_user %}
+    {% set user_logged_in = current_user.is_authenticated() if current_user.is_authenticated is callable else current_user.is_authenticated %}
+  {% endif %}
+
+  {% if user_logged_in %}
+    {% set headerNavigation = headerNavigation + [{
+      'text': '<form method="post" action="' + url_for('external.user_logout') + '">
+              <input type="submit" class="logout-button" value="Log out" />
+              <input type="hidden" name="csrf_token" value="' + (csrf_token_value or csrf_token()) + '" />
+            </form>',
+      'href': url_for('external.render_login')
+    }]%}
+  {% else %}
+    {% set headerNavigation = headerNavigation + [{
+      'text': 'Log in',
+      'href': url_for('external.render_login')
+    }]%}
+  {% endif %}
```

Comment:
Is this even needed for admin frontend? Users can only use Admin frontend if they are logged in else they are redirected to user-frontend to authenticate.

Reply:
No theres no reason for us to check if the user is logged in or not.